src/App.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
+import './app.scss';
+import { Button } from 'carbon-components-react';
```

Comment:
The button import statement should be removed since the button component is not used.